gemini/getting-started/intro_gemini_2_5_pro.ipynb

```diff
@@ -379,7 +379,7 @@
         "**Notes**\n",
         "\n",
         "- By default, the model automatically controls how much it thinks up to a maximum of 8192 tokens.\n",
-        "- The maximum thinking budget that you can set is 24576 tokens, and the minimum you can set is 64.\n",
+        "- The maximum thinking budget that you can set is 32768 tokens, and the minimum you can set is 128.\n",
```

Comment:
You can set the `THINKING_BUDGET` param on line 395 to have a specific range with

```
THINKING_BUDGET = 1024 # @param {type: "integer", min: 128, max: 32768}
```